### DISCUSSION TITLE
What's next: the Fluent UI Blazor library v5

### DISCUSSION THREAD

**Discussion by vnbaaij** [microsoft/fluentui-blazor#3066] [Announcements]
Hello all!

I've posted a blog about the next major version of the library and how we see the support policy for the v4 version, and how we think about upgrading v4 environments. Please see https://baaijte.net/blog/microsoft-fluentui-aspnetcore.components-5/ and use the discussion here to exchange thoughts about this with us. We look forward to a fruitful discussion with you all.

Oh, and if you are celebrating, happy holidays!! 🎄🎅🏾

**Comment by MarvinKlein1508** [comment]
Great article @vnbaaij !

do you think you could utilize the Project feature of GitHub to track progress for v5? 

I would love to see which stuff is being worked on or has been completed already for the new version. Also it would be nice to have the link for the preview site somewhere: [https://fluentui-blazor-v5.azurewebsites.net](https://fluentui-blazor-v5.azurewebsites.net/MigrationV5)

**Reply by vnbaaij** [reply]
> do you think you could utilize the Project feature of GitHub to track progress for v5?

That would imply that we have a sort of roadmap 😉...

We can take a look at that in a couple of weeks. We need to figure out the sequence of doing things first. Busy with v4.11...

**Reply by MarvinKlein1508** [reply]
Yeah take your time. Enjoy the holidays first! :) 

I try to help you guys out as much as I can. This library has really changed my development productivity and I always learn new stuff here when I investigate some of the open issues! :) Used so many techniques from this library already in my project! :)

**Reply by dvoituron** [reply]
> I try to help you guys out as much as I can. This library has really changed my development productivity and I always learn new stuff here when I investigate some of the open issues! :) Used so many techniques from this library already in my project! :)

Thanks 😀

**Comment by AlexHedley** [comment]
I think the link is broken:
> We have started work on this already and progress can be followed in the `dev-v5` branch in our [repo](https://github.com/fluentui-blazor).

Oh, I meant to just link to here and then people can go to the branch themselves.

**Reply by AlexHedley** [reply]
https://github.com/microsoft/fluentui-blazor/tree/dev-v5

**Comment by parsalotfy** [comment]
when will this version 5 be released?🚀

**Reply by vnbaaij** [reply]
We don't know yet. Web Components are still in beta and there is still **_a lot_** that needs to be done by us.

(we never communicate exact dates on releases)
